packages/sourcecred/src/core/credGrainView.js

```diff
@@ -37,6 +37,8 @@ import * as C from "../util/combo";
  * TimeScopedCredGrainView that generated this.
  */
 export type ParticipantCredGrain = {|
+  +active: boolean,
+  // Is the user active
```

Comment:
Happy to get rid of it, but was following convention. Here is the whole block:

export type ParticipantCredGrain = {|
  +active: boolean,
  // Is the user active
  +identity: Identity,
  // Total Cred earned during the time scope.
  +cred: number,
  // Cred earned in each interval within the time scope.
  +credPerInterval: $ReadOnlyArray<number>,
  // Total Grain earned during the time scope.
  +grainEarned: Grain,
  // Grain earned in each interval within the time scope.
  +grainEarnedPerInterval: $ReadOnlyArray<Grain>,
|};

Reply:
ah fair. Well if kept, it should go above the line it describes instead of below, and be more descriptive like "is the user activated for grain distributions"